xc/xc7/archs/zynq7_z020/tiles/clbll_l/CMakeLists.txt

```
@@ -1,5 +1,5 @@
 project_ray_tile(
```

The cmake function should be changed as follows (for all CLBs)

```suggestion
project_ray_tile_capacity(
```